src/grammar/grammar.ohm

```diff
@@ -215,8 +215,17 @@ Tact {
     boolLiteral = ("true" | "false") ~idPart
 
     // String literal
-    stringLiteralCharacter = ~("\"" | "\\" | lineTerminator) any
-    stringLiteral = "\"" stringLiteralCharacter* "\""
+    // stringLiteralCharacter = ~("\"" | "\\" | lineTerminator) any
+    // stringLiteral = "\"" stringLiteralCharacter* "\""
+    stringLiteral = "\"" (nonQuoteOrBackslashChar | escapeSequence)* "\""
```

Comment:
Let's support all the things from the lexical grammar of JS: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#string_literals.

It has more escape sequences including Unicode, and also more ASCII things, like `\v`, etc. And it allows to split long strings across multiple lines.

Reply:
For an example of this in Ohm, check out [`ohm-grammar.ohm`](https://github.com/novusnota/vscode-ohm/blob/main/samples/ohm-grammar.ohm#L93-107)